docs/user/index.rst

```diff
@@ -0,0 +1,120 @@
+######################
+Using Aldryn Reversion
+######################
+
+After you set up and configured your models (they are registered for revisions
+support) you can start using ``aldryn-reversion`` end-user features.
+
+************************
+Restoring object version
+************************
+
+Most likely you already know that django keeps track of object changes also
+known as history, it is available on object admin edit form on the top of the
+page. With ``aldryn-reversion`` that history has more features.
+As soon as object is created - 'Initial version' is created and treated as
+restore point.
+
+Each record in history has a date which is now a link to restore revision
+form.
+When you need to restore any object version you just need to click on desired
+date, check what will be changed  and either restore that revision or go back
+to object history.
+
+.. note::
+    Be aware that in current implementation revision is restored as is.
+    That means that any related object that is also stored in this revision
+    would be restored at the point at which that object was stored.
+    This is applicable to whole relation tree (if follow was configured).
+
+As soo as you restore object version the object would be at the version at
+which it was kept and you will be redirected to object edit form so that you
+can change object if you need to.
+
+You might also want to change related object (if applicable) since they also
+will be affected.
+
+.. note::
+   If :ref:`follow` was not configured properly and related object was not
```

Comment:
How about:
"If :ref:`follow` was not properly configured and as a result the related objects were not stored as part of the selected revision, you could end up with an ``IntegrityError``. In such a case you will first need to restore the related object."
?